db/blip_handler.go

```diff
@@ -242,7 +242,7 @@ func (bh *blipHandler) handleSetCheckpoint(rq *blip.Message) error {
 func (bh *blipHandler) handleSubChanges(rq *blip.Message) error {
 	defaultSince := CreateZeroSinceValue()
 	latestSeq := func() (SequenceID, error) {
-		seq, err := bh.collection.LastSequence()
+		seq, err := bh.collection.LastSequence(bh.loggingCtx)
```

Comment:
As mentioned via Slack - What looks like a 3.1.0 regression - `bh.loggingCtx` isn't being set properly for default collection replications, resulting in us losing context ID for a lot of logging called via BLIP.